app/services/service_provider_request_proxy.rb

```diff
@@ -78,8 +78,7 @@ def self.flush
   end
 
   def self.hash_to_spr(hash, uuid)
-    spr = ServiceProviderRequest.new(hash)
-    spr.uuid = uuid
-    spr
+    hash ||= { issuer: nil, url: nil, loa: nil, requested_attributes: nil }
```

Comment:
when is hash `nil`? Is the goal here to provide default attributes? If so, you could do:

```ruby
hash = { issuer: nil, url: nil, loa: nil, requested_attributes: nil }.merge(hash)
```

(there's also `reverse_merge` but I think that is harder to read)

Reply:
good catch it won't be nill.  this was leftover when my first cut was using OpenStruct